lib/fluent/plugin/out_http.rb

```diff
@@ -302,16 +318,41 @@ def create_request(chunk, uri)
       req
     end
 
+    def make_request_cached(uri, req)
+      id = Thread.current.thread_variable_get(plugin_id)
```

Comment:
It's not appropriate to use `plugin_id` as a key, it should be used with a suffix like "#{plugin_id}_connection_cache_id".

In addition, we prefer to use `Thread#[]` instead of `Thread#thread_variable_get` if there is no particular reason.

Reply:
Thanks. I agree.
Done in 9ce635ce524a30622267b5686d9a53ab00269d1c.